src/main/java/Loopy.java

```diff
@@ -0,0 +1,133 @@
+            processTask(task);
+            if (task.equals("bye")) {
+                System.out.println("Bye! Hope to see you again.\n");
+                System.out.println(".                       \n" +
+                        "                                 --                      \n" +
+                        "                                 =..                     \n" +
+                        "                                .=.::                    \n" +
+                        "                                :=:.                     \n" +
+                        "                 .::.. ....::::::::..                    \n" +
+                        "                .-+++-:::::::::::::::::..                \n" +
+                        "                 :+=-:::::----------::::::....           \n" +
+                        "                 .------------------------:=++-.         \n" +
+                        "                .---------------------------=+=.         \n" +
+                        "                :---------===-------=-------=:.          \n" +
+                        "               .---------===================-            \n" +
+                        "              .--------=#%#*+===============-            \n" +
+                        "             .----------*%%+======++===----.           \n" +
+                        "             :=------------------=+@@#=------.           \n" +
+                        "             -======---==-=%@@@%*---==-------.           \n" +
+                        "             -+===========*@@@%*##====-------.           \n" +
+                        "             .+++++++++++++#%@@@#++==========.           \n" +
+                        "              :+++++++*********+++=========.           \n" +
+                        "               .=********##=-=*#**++++++++++:            \n" +
+                        "                  -+#******++******++++++:             \n" +
+                        "            .:-=-.:**###****************+.               \n" +
+                        "          .=++***=++****#########+=--:..                 \n" +
+                        "         .=+****===+++++*********+:                      \n" +
+                        "         .++***===+*+++++++++++++++-.                    \n" +
+                        "          =+++-===+*++=====----==+*+=.                   \n" +
+                        "          .++---=+*+===-:::::::---++:                  \n" +
+                        "           -----=+*++=::::::::::-:=*+++=.                \n" +
+                        "            :-=-##*+=-::::::::::::-+++=.               \n" +
+                        "            :%#*#+=--:::::::::::-.:=*%#+               \n" +
+                        "             -++++++----:::::::::--.  .:-.");
+                break;
+            }
+        }
+    }
+
+    private static void processTask(String task) {
```

Comment:
I like how you followed the naming convention and wrote the variable name in camelCase. (the loopy is so nicee!!!)